scripts/prepare_systemd_network.sh

```diff
@@ -19,3 +21,30 @@ EOF
 mkdir -p "$WORKING_DIR/etc/systemd/resolved.conf.d"
 
 cp "$TEMPLATES_DIR/cluster.local.conf" "$WORKING_DIR/etc/systemd/resolved.conf.d/"
+
+# If in running in Azure, then create the two network files ".netdev" and ".network" required by the new VXLAN
+if [ $IS_IN_AZURE == "true" ]; then
+    IFS=, read -ra MASTERS <<< "$MASTER_VIP_CLUSTER_IPS"
+
+    cp "$TEMPLATES_DIR/vxlan0.netdev" "$OUTPUT_DIR/vxlan0.netdev"
+
+    echo "
+[Match]
+Name=vxlan0
+
+[Network]
+Address=$VXLAN_IP
+" > $OUTPUT_DIR/vxlan0.network
+
+    for ((i=0; i<${#MASTERS[@]}; i++)); do
+        if [ ${MASTERS[i]} = $NODE_HOST_IP ]; then
+            continue
+        fi
+
+        echo "
+[BridgeFDB]
+MACAddress=00:00:00:00:00:00
+Destination=${MASTERS[i]}
+" >> $OUTPUT_DIR/vxlan0.network
+    done
```

Comment:
Indentation of the for loop

```suggestion
  for ((i=0; i<${#MASTERS[@]}; i++)); do
      if [ ${MASTERS[i]} = $NODE_HOST_IP ]; then
          continue
      fi

      echo "
[BridgeFDB]
MACAddress=00:00:00:00:00:00
Destination=${MASTERS[i]}
" >> $OUTPUT_DIR/vxlan0.network
  done
```